TS01-Time-Series_Metadata.Rmd

```diff
@@ -65,37 +60,20 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-#Understand Your Data 
```

Comment:
@lwasser said that most people who access the existing online NEON tutorials do so via  a search that lands them on a particular page, not the full workshop/ module.  I can say that as I've been figuring out POSIX and other items I haven't used before I'm doing the same thing.  Looking for the key information on a page and then leaving, only sometimes clicking to linked pages if it seems like the information I need is there. 

I think for these lessons we 1) allow for a bit of overlap or repetitive information in the top section where the data needs to be loaded, 2) keep the overlap to background concept (e.g. why these variables) that is needed to understand the very basics of the lesson, not background skills,  3) link to previous lessons when it is skills/ a longer explanation that is needed and finally 4) each lesson's R script should be able to fully function without needing any R objects, etc from a previous lesson (except for the working directory compromise we've added in).  

With that in mind would you mind merging the deleted info in this top section with your edits?